kuma/banners/jinja2/banners/banners.html

```diff
@@ -0,0 +1,52 @@
+{% set banners = list(get_banners()) %}
+<div
+    id="developer_needs"
+    class="mdn-cta-container hidden"
+    aria-expanded="false"
+    aria-hidden="true"
+    data-banner="developer_needs"
+>
+    {% if banners %}
+        {% for banner in banners %}
+            <div class="mdn-cta-content {{ banner.theme }}">
+                <div id="developer-needs" class="mdn-cta-content-container">
```

Comment:
You mentioned you were planning to edit all of this. And I'm not the reviewer. But I don't think the PR is too big or noisy to include your other changes for this. 
Right now it's dangerously wrong with the ID being repeated in a for-loop.

Reply:
Ah yes. I am aware of that. But as of right now, and this won't change soon, there will only be one banner.

With that said, in the follow on PR, I will add support for multiple banners as that is the long term use case for this.

Reply:
What is the follow-on PR going to be about? Is there a reason you can't do it as part of this PR. Is there benefits in landing this (with its little flaws) and having it in prod for a while? 

One concern I have is that the l10n'ness of the banner's content is far from worked out and unless there's an urgent rush I think we should nail that *and* this niggled about repeated IDs.

Reply:
By the way, why does it need an ID at all??